frontend/apps/marketing/src/components/tabGroup/TabGroup.tsx

```diff
@@ -0,0 +1,89 @@
+'use client';
+import {BaseEntry} from 'contentful';
+import {useMemo} from 'react';
+
+import TabGroup, {
+  TabGroupTabModel,
+} from '@code-dot-org/component-library/cms/tabGroup';
+
+import {externalLinkIconProps} from '@/components/common/constants';
+
+type TabGroupContentfulProps = {
+  tabs?: (BaseEntry & {
+    fields: {
+      ctaLink: BaseEntry & {
+        fields: {
+          ariaLabel?: string;
+          isThisAnExternalLink: boolean;
+          label: string;
+          primaryTarget: string;
+        };
+      };
+      description: string;
+      image: BaseEntry & {
+        fields: {
+          description?: string;
+          title?: string;
+          file: {
+            url: string;
+          };
+        };
+      };
+      tabLabel: string;
+      title: string;
+    };
+  })[];
+};
+
+const TabGroupContentful: React.FunctionComponent<TabGroupContentfulProps> = ({
+  tabs = [],
+}) => {
+  const parsedTabs: TabGroupTabModel[] = useMemo(
+    () =>
+      tabs.map(tab => ({
+        value: tab.fields.tabLabel,
+        text: tab.fields.tabLabel,
+        tabContent: {
+          title: tab.fields.title,
+          description: tab.fields.description,
+
+          image: {
+            src: `https:${tab.fields.image.fields?.file.url}`,
+            alt: tab.fields.image.fields?.description,
+          },
+          button: {
+            href: tab.fields.ctaLink.fields?.primaryTarget || '#',
+            text: tab.fields.ctaLink.fields?.label || '#',
+            iconRight: tab.fields.ctaLink.fields?.isThisAnExternalLink
+              ? externalLinkIconProps
+              : undefined,
+          },
+        },
+      })),
+    [tabs],
+  );
+
+  // Show placeholder text until a content entry is added
+  if (!tabs.length) {
+    return (
+      <div style={{color: 'var(--text-neutral-primary)'}}>
+        <em>
+          <strong>❓ Tab Group placeholder.</strong> Please add a "Tabs" content
```

Comment:
Nit: update this to match the Tab Group content type emoji
```suggestion
          <strong>🗂️ Tab Group placeholder.</strong> Please add a "Tabs" content
```